UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Utilities/FileUtility.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using System.Security.Cryptography;
+using DiscUtils.Iso9660;
+namespace UKHO.PeriodicOutputService.Common.Utility
+{
+    [ExcludeFromCodeCoverage]
+    public class FileUtility : IFileUtility
+    {
+        public void CreateISOImage(IEnumerable<string> srcFiles, string targetPath, string directoryPath)
+        {
+            var iso = new CDBuilder
+            {
+                UseJoliet = true,
+                VolumeIdentifier = "FullAVCSExchangeSet"
+            };
+
+            foreach (string? file in srcFiles)
+            {
+                var fileInfo = new FileInfo(file);
+                if (fileInfo.Directory.Name == directoryPath)
+                {
+                    iso.AddFile($"{fileInfo.Name}", fileInfo.FullName);
+                    continue;
+                }
+                string? srcDir = fileInfo.Directory.FullName.Replace(directoryPath, "").TrimEnd('\\');
```

Comment:
srcDir doesn't need to be nullable, so you can remove the ?

Reply:
Removed nullable